<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="com.creativity.dev.formsimple.ToolsUnitTest" tests="4" skipped="0" failures="1" errors="0" timestamp="2024-01-02T20:57:05" hostname="Tropigass-MacBook-Pro.local" time="41.813">
  <properties/>
  <testcase name="test_validateByTag_isCorrect" classname="com.creativity.dev.formsimple.ToolsUnitTest" time="41.017">
    <failure message="org.junit.ComparisonFailure: expected:&lt;[OK]&gt; but was:&lt;[Error]&gt;" type="org.junit.ComparisonFailure">org.junit.ComparisonFailure: expected:&lt;[OK]&gt; but was:&lt;[Error]&gt;
	at org.junit.Assert.assertEquals(Assert.java:117)
	at org.junit.Assert.assertEquals(Assert.java:146)
	at com.creativity.dev.formsimple.ToolsUnitTest.test_validateByTag_isCorrect$lambda$2(ToolsUnitTest.kt:62)
	at androidx.test.core.app.ActivityScenario.lambda$onActivity$2$androidx-test-core-app-ActivityScenario(ActivityScenario.java:789)
	at androidx.test.core.app.ActivityScenario$$ExternalSyntheticLambda2.run(D8$$SyntheticClass)
	at androidx.test.core.app.ActivityScenario.onActivity(ActivityScenario.java:799)
	at com.creativity.dev.formsimple.ToolsUnitTest.test_validateByTag_isCorrect(ToolsUnitTest.kt:57)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.robolectric.RobolectricTestRunner$HelperTestRunner$1.evaluate(RobolectricTestRunner.java:588)
	at org.robolectric.internal.SandboxTestRunner$2.lambda$evaluate$2(SandboxTestRunner.java:290)
	at org.robolectric.internal.bytecode.Sandbox.lambda$runOnMainThread$0(Sandbox.java:101)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
</failure>
  </testcase>
  <testcase name="test_validateAll_isCorrect" classname="com.creativity.dev.formsimple.ToolsUnitTest" time="0.047"/>
  <testcase name="test_init_isCorrect" classname="com.creativity.dev.formsimple.ToolsUnitTest" time="0.719"/>
  <testcase name="addition_isCorrect" classname="com.creativity.dev.formsimple.ToolsUnitTest" time="0.03"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[No package ID ff found for ID 0xffffff00.
No package ID ff found for ID 0xffffff00.
No package ID ff found for ID 0xffffff00.
No package ID ff found for ID 0xffffff00.
No package ID ff found for ID 0xffffff00.
No package ID ff found for ID 0xffffff00.
No package ID ff found for ID 0xffffff00.
No package ID ff found for ID 0xffffff00.
No package ID ff found for ID 0xffffff00.
No package ID ff found for ID 0xffffff00.
No package ID ff found for ID 0xffffff00.
No package ID ff found for ID 0xffffff00.
No package ID ff found for ID 0xffffff00.
No package ID ff found for ID 0xffffff00.
]]></system-err>
</testsuite>
